src/bilder/images/edxapp/templates/edxapp/mitx-staging/common_values.yml

```diff
@@ -457,7 +457,7 @@ SENTRY_ENVIRONMENT: {{ env "ENVIRONMENT" }}
 SERVER_EMAIL: odl-devops@mit.edu # MODIFIED
 # Removing the session cookie domain as it is no longer needed for sharing the cookie
 # between LMS and Studio (TMM 2021-10-22)
-# SESSION_COOKIE_DOMAIN: {{ key "edxapp/cookie-domain" }} # MODIFIED
+# SESSION_COOKIE_DOMAIN:  # MODIFIED
```

Comment:
Do we need to keep the key if it has no value set?

Reply:
Most likely not, but I'm being conservative while we work through this deployment. It also acts as a reminder to dig into it when we have the time to trim down configuration settings even further.